utils/utils.go

```diff
@@ -53,6 +53,7 @@ func GetRevision() string {
 
 func init() {
 	if runtime.GOOS == "windows" {
+		ViamDirs["viam"] = "c:/opt/viam"
```

Comment:
Is it always c:? I thought we had concerns about this

Reply:
No, it's probably not... but previously we were just going with "random drive" and not even specifying it, so that was worse.

Likely needs to look for some ENV var to get the WINDOWS_PATH or something, and determine the system drive from there. Had meant to pursue further.

Reply:
One further thought... this is correct for at least as long as we're hardcoding that same path/drive in the installer (which it looks like we are.) When we fix this, we really should be putting it in %PROGRAM_FILES% or %APPLICATION_DATA% and stuff, not unix-style paths like "/opt" anyway.